docs/io/unified.rst

```diff
@@ -322,6 +331,140 @@ FITS time standard will be set.  When reading this back into Astropy, the
+
+Time coordinate columns strictly compliant with the two-vector JD subset of the
+standard (described in the `Details`_ section below) can be read as native
+`~astropy.time.Time` objects. The other subsets of the standard are also supported
+by Astropy; a thorough examination of the FITS standard time-related keywords is
+done and the time data is interpreted accordingly.
+
+The standard describes the various components in the specification of time:
+
+- Time coordinate frame
+- Time unit
+- Corrections, errors, etc.
+- Durations
+
+The keywords used to specify times define these components. Using these keywords,
+time coordinate columns are identified and read as `~astropy.time.Time` objects.
+Refer to :ref:`fits_time_column` for the specification of these keywords and their
+description.
+
+There are two aspects of the standard that require special attention due to the
+subtleties involved while handling them. These are:
+
+* Column named TIME with time unit
+
+A common convention found in existing FITS files is that a FITS binary
+table column with ``TTYPEn = ‘TIME’`` represents a time coordinate column.
+Many astronomical data files, including official data products from major
+observatories, follow this convention that pre-dates the FITS standard.
+The FITS time standard states that such a column will be controlled by
+the global time reference frame keywords, and this will still be compliant
+with the present standard.
+
+Using this convention which has been incorporated into the standard, Astropy
+can read time coordinate columns from all such FITS tables as native
+`~astropy.time.Time` objects. Common examples of FITS files following
+this convention are Chandra, XMM, and HST files.
+
+The following is an example of a Header extract of a Chandra event list:
+
+.. parsed-literal::
+
+    COMMENT      ---------- Globally valid key words ----------------
+    DATE    = '2016-01-27T12:34:24' / Date and time of file creation
+    TIMESYS = 'TT      '           / Time system
+    MJDREF  =  5.0814000000000E+04 / [d] MJD zero point for times
+    TIMEUNIT= 's       '           / Time unit
+    TIMEREF = 'LOCAL   '           / Time reference (barycenter/local)
+
+    COMMENT      ---------- Time Column -----------------------
+    TTYPE1  = 'time    '           / S/C TT corresponding to mid-exposure
+    TFORM1  = '1D      '           / format of field
+    TUNIT1  = 's       '
+
+When reading such a FITS table with ``astropy_native=True``, Astropy checks
+whether the name of a column is "TIME"/ "time" (``TTYPEn = ‘TIME’``) and
+whether its unit is a FITS recognized time unit (``TUNITn`` is a time unit).
+
+For example, reading a Chandra event list which has the above mentioned header
+and the time coordinate column ``time`` as ``[1, 2]`` will give::
+
+    >>> from astropy.time import Time, TimeDelta
+    >>> from astropy.table import Table
+    >>> tm = Table.read('chandra_events.fits', astropy_native=True)
```

Comment:
Since we ship a sample chandra events file, would it be an idea to use that here? That way the doctest could actually be run.

Reply:
I just have to add a relative path to the file to access it here, right? (and also I would have to remove doctest-skip-all)